contribs/gmf/src/directives/search.js

```diff
@@ -105,6 +118,14 @@ gmf.SearchController = function($scope, $compile,
   this.map_ = map;
 
   /**
+   * Use or not a clear button to clear the search's dropdown.
```

Comment:
This doesn't really clears the dropdown. This is a consequence of clearing the text input. I'd use the following:
"Whether or not to show a button to clear the search text."